src/devices/cpu/hd61700/hd61700.h

```diff
@@ -69,24 +68,26 @@ class hd61700_cpu_device : public cpu_device
 	// interrupts
 	bool check_irqs();
 
-	// inline helpers
-	inline void  set_pc(int32_t new_pc);
-	inline uint8_t read_op();
-	inline uint8_t mem_readbyte(uint8_t segment, uint16_t offset);
-	inline void  mem_writebyte(uint8_t segment, uint16_t offset, uint8_t data);
-	inline uint32_t make_18bit_addr(uint8_t segment, uint16_t offset);
-	inline int   check_cond( uint32_t op );
-	inline void  push(uint16_t &offset, uint8_t data);
-	inline uint8_t pop(uint16_t &offset);
-	inline uint8_t make_logic(uint8_t type, uint8_t d1, uint8_t d2);
-	inline void  check_optional_jr(uint8_t arg);
-	inline uint8_t get_sir_im8(uint8_t arg);
-	inline uint8_t get_sir_im8(uint8_t arg, uint8_t arg1);
-	inline int   get_sign_mreg(uint8_t op1);
-	inline int   get_sign_im8(uint8_t op1);
-	inline int   get_im_7(uint8_t data);
-	inline uint16_t make_bcd_sub(uint8_t arg1, uint8_t arg2);
-	inline uint16_t make_bcd_add(uint8_t arg1, uint8_t arg2);
+	// helpers
+	void     set_pc(int32_t new_pc);
+	uint8_t  read_op();
+	uint8_t  mem_readbyte(uint8_t segment, uint16_t offset);
+	void     mem_writebyte(uint8_t segment, uint16_t offset, uint8_t data);
+	uint32_t make_18bit_addr(uint8_t segment, uint16_t offset);
+	int      check_cond( uint32_t op );
+	void     push(uint16_t &offset, uint8_t data);
+	uint8_t  pop(uint16_t &offset);
+	uint8_t  make_logic(uint8_t type, uint8_t d1, uint8_t d2);
+	void     check_optional_jr(uint8_t arg);
+	uint8_t  get_sir_im8(uint8_t arg);
+	uint8_t  get_sir_im8(uint8_t arg, uint8_t arg1);
+	int      get_sign_mreg(uint8_t op1);
+	int      get_sign_im8(uint8_t op1);
+	int      get_im_7(uint8_t data);
+	uint16_t make_bcd_sub(uint8_t arg1, uint8_t arg2);
+	uint16_t make_bcd_add(uint8_t arg1, uint8_t arg2);
```

Comment:
This looks like another case where `inline` is being used to tell the compiler it doesn’t need to generate out-of-line bodies – removing `inline` from the declarations in the header is harmless, but I don’t think removing it from all the implementations is a good idea.

Reply:
inline re-added to the .cpp file.